backend/src/layer/logic/mealplan_management/relation_resolver.rs

```diff
@@ -1,9 +1,32 @@
 use crate::interface::mensa_parser::model::{Dish, ParseCanteen, ParseLine};
 use crate::interface::persistent_data::{DataError, MealplanManagementDataAccess};
 use crate::util::{Date, Uuid};
+use std::collections::HashMap;
 use std::slice::Iter;
+use lazy_static::lazy_static;
 use tracing::warn;
 
+lazy_static! {
+    static ref CANTEEN_POSITIONS: HashMap<&'static str, u32> = HashMap::from([
```

Comment:
is it possible to not hard code thah but rather infer it from the to the parser provided list of canteens?
We then probably have to do that in the parse componennt

Reply:
deleted